stat_var_renaming/pop_obs_spec_nocovid.textproto

```diff
@@ -0,0 +1,1707 @@
+## Person Count ##
```

Comment:
How was this created?

Reply:
The original is here: https://github.com/datacommonsorg/website/blob/4d6f62201337a0603e781a571ca6e09de22a18e2/tools/pv_tree_generator/pop_obs_spec_common.textproto

@IanCostello  can you make a comment in the README about how you separated into the two file here?